app/helpers/notification_mailer_helper.rb

```diff
@@ -51,10 +51,10 @@ def new_group_body(data)
       val('new_group_body'),
       user: data['user'].to_s,
       group: data['group'].to_s,
-      description: Group.where(id: data['group_id']).first.description,
+      description: Group.find_by(id: data['group_id']).description,
```

Comment:
👍 I hope there will always be a group with this id `data['group_id']` otherwise we will have to deal with `no method error description for nil class`

Reply:
There should be, but I can double check and fix it in another PR (since it's technically out of scope). Good call out, thanks :)